packages/core/utils.js

```diff
@@ -13,37 +13,29 @@ export const extractComponentFromChildren = (_children, componentName) => {
   return [children, component];
 };
 
-export const buildTheme = (components, base) => {
-  // Resolve references to base theme throughout component themes
-  // eg, `link.space.mx: 2` -> `link.space.mx: '1.6rem'`
-  // eg, `link.colors.hoverColor: 'primary'` -> `link.colors.hoverColor: '#738FFC'`
+export const buildTheme = (base, components) => {
+  let baseTheme = { ...base };
 
-  let theme = { ...base };
   for (var component in components) {
     const componentTheme = components[component];
-    for (var themeCategory in componentTheme) {
-      if (["icons", "__filemeta"].includes(themeCategory)) continue;
-      // lest we want to resolve icon names with their SVG strings
+    replaceThemeRefs(componentTheme, baseTheme);
+    baseTheme[component] = componentTheme;
+  }
+  console.log("Base theme after stuff", baseTheme);
```

Comment:
Did you want to leave this `console.log` statement in?

Reply:
For the time, I do. It incredibly helpful during development